pkg/cmd/roachtest/cluster.go

```diff
@@ -2299,8 +2300,10 @@ func (c *clusterImpl) ExternalIP(
 var _ = (&clusterImpl{}).ExternalIP
 
 // Conn returns a SQL connection to the specified node.
-func (c *clusterImpl) Conn(ctx context.Context, l *logger.Logger, node int) *gosql.DB {
-	urls, err := c.ExternalPGUrl(ctx, l, c.Node(node))
+func (c *clusterImpl) Conn(
+	ctx context.Context, l *logger.Logger, node int, tenant string,
```

Comment:
I wonder if this might be a place where we want to use the "functional options" pattern so that we could do something like:

```golang
func (c *clusterImpl) Conn(ctx context.Context, l *logger.Logger, node int, opts ...ConnectionOption)
```

such that we don't have to update every single caller.

Reply:
aha, this is a good idea. I'll plan to refactor with `...ConnectionOptions` pattern as it will take 5 minutes to refactor with Goland's fancy refactor tool. I'll wait until @smg260 and/or @renatolabs comment on this API change though.

Reply:
Definitely a 👍 from me.

Reply:
function options are _slick_